md/release-notes.md

```diff
@@ -41,19 +53,27 @@ With the autocomplete widget you have a returned value that different from the d
 
 ### Runtime changes
 
-#### Micrometer updated to 1.6.0
```

Comment:
Shouldn't that be part of the Technical updates below?

Reply:
It explain the new metric added and that we contributed to this release